spring-boot-starter-camunda/src/main/java/io/camunda/common/http/Java8Utils.java

```diff
@@ -0,0 +1,37 @@
+package io.camunda.common.http;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class Java8Utils {
+
+  private Java8Utils() {}
```

Comment:
I guess spring-zeebe was supporting java 8, let's verify and follow-up if that is still needed

Reply:
Given spring-zeebe 8.4+ already required jdk 17 I guess it's not needed anymore https://github.com/camunda-community-hub/spring-zeebe